tephra-core/src/test/java/org/apache/tephra/ThriftTransactionSystemTest.java

```diff
@@ -88,10 +89,14 @@ protected void configure() {
     txClient = injector.getInstance(TransactionSystemClient.class);
     try {
       LOG.info("Starting transaction service");
+      storage.startAndWait();
```

Comment:
why start this here, but not in other test cases (TransactionAdminTest, for instance)?